torch/onnx/symbolic_opset9.py

```diff
@@ -1908,6 +1908,12 @@ def multinomial(g, input, num_samples, replacement=False, generator=None):
                 dtype_i=sym_help.cast_pytorch_to_onnx['Long'],
                 sample_size_i=num_samples)
 
+def baddbmm(g, self, batch1, batch2, beta, alpha):  
```

Comment:
Can this be exported as onnx::Gemm?

Reply:
In onnx::Gemm, alpha and beta are attributes. So we cannot use them as dynamic inputs.